core/trino-main/src/main/java/io/trino/operator/project/PageProcessorMetrics.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.operator.project;
+
+import com.google.common.collect.ImmutableMap;
+import io.airlift.units.Duration;
+import io.trino.plugin.base.metrics.DurationTiming;
+import io.trino.spi.metrics.Metric;
+import io.trino.spi.metrics.Metrics;
+
+import static java.util.concurrent.TimeUnit.NANOSECONDS;
+
+public class PageProcessorMetrics
+{
+    private static final String FILTER_TIME = "Filter execution time";
+    private static final String PROJECTION_TIME = "Projection time";
```

Comment:
Why the asymmetry in the names? Also, does the measurement include potential I/O or is it purely CPU time? If the latter, call it "xxx CPU time". Otherwise, the could be named "xxx Wall time"

Reply:
This should measure only CPU and not I/O. I've renamed it to "Filter CPU time" and "Projection CPU time".